src/models/Factor.js

```diff
@@ -255,6 +255,14 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
       // we don't need passcode for email with magic link flow
       return !(this.options.appState.get('isIdxStateToken') && this.get('factorType') === 'email');
     },
+    resend: function () {
```

Comment:
Why was resend added here?

Reply:
Because email enroll and email verification are using same `ResendEmailView.js` to render resend view, which trigger resend by invoking `model.resend`.
However email enroll and email verification are using different Model, EnrollActivateEmailController.Model v.s. Factor.js hence need to update the Factor.js